airflow/www_rbac/views.py

```diff
@@ -0,0 +1,2633 @@
+# -*- coding: utf-8 -*-
```

Comment:
I am not sure if we have added all this logic. @vshshjn7 can you please check and confirm?